src/style.css

```diff
@@ -1,46 +1,112 @@
 body {
-  background-color:#f3f4f6;
+  background-color: #f3f4f6;
 }
 
 .nav,
 .content-cont,
-.inner-comments {
+.inner-comments,
+.comments-div,
+.like-div {
   justify-content: center;
   align-items: center;
 }
 
-.content-cont {
-  flex-wrap: wrap;
+.btn-nav {
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.nav {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  height: 80px;
+  background-color: gray;
+}
+
+.btn-save {
+  width: 50%;
+}
+
+.active {
+  background-color: rgb(0, 0, 0.5);
+  position: fixed;
+  opacity: 0.5;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
+
+.like-div {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.like-div button {
+  border: none;
+  outline: none;
+  color: red;
+  font-weight: 500;
+  font-size: 1.2rem;
+}
+
+.like-div button span {
+  color: #000;
+}
+
+.add-comment,
+.comment-count {
+  font-weight: bold;
 }
 
-.reserve-form,
-.comments-form {
-  position: absolute;
-  top: 50px;
-  left: 10%;
-  right: 10%;
-  width: 80%;
-  height: 100%;
+.name-title {
+  font-weight: 500;
+}
+
+.popup-form {
+  display: none;
+  position: fixed;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+  width: 600px;
+  height: 80vh;
+  margin: auto;
   box-shadow: 2px 2px 5px black;
   background-color: #fff;
   padding: 10px 20px;
-  display: none;
   z-index: 500;
+  overflow-y: scroll;
 }
 
-img {
-  width: 100%;
+.comments-div {
+  flex-wrap: wrap;
+  margin: 0 20px;
+  margin-bottom: 5%;
 }
```

Comment:
[Optional]
- [ ] Instead of using flexbox for this section, you can use a grid and create a better interface. 

If you want to apply this suggestion, don't forget to remove the `d-flex` class from the container `div`

With Flexbox:
![Screenshot 2021-08-07 at 20-14-10 capstone](https://user-images.githubusercontent.com/1642014/128608974-d36c4df5-7bbb-4795-bd32-2d050d20d57c.png)

With Grid:
![Screenshot from 2021-08-07 20-28-41](https://user-images.githubusercontent.com/1642014/128608959-64b8676b-7c0f-4179-ad17-10de2a341724.png)


```suggestion
.comments-div {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-gap: 20px;
}
```